### ISSUE TITLE
Nonovar dose not parse -x correctly when utilizing minimap2

### ISSUE THREAD

**Issue by LYC-vio** [cytham/nanovar#36]
While NanoVar uses:
```
-x str, --data_type str
                        type of long-read data [ont]
                        ont - Oxford Nanopore Technologies
                        pacbio-clr - Pacific Biosciences CLR
                        pacbio-ccs - Pacific Biosciences CCS
``` 
to set the datatype, the minimap2 part inside `nanovar/nv_align.py` only has if-else for data_type == 'ont' and data_type == 'pacbio' (see below), which means datatype `pacbio-clr` and `pacbio-ccs` won't be parsed correctly, resulting in ` ... -ax ref.fa ... `  -> minimap preset error ` [ERROR] unknown preset 'ref.fa' `
```
def align_mm(ref, read, wk_dir, read_name, ref_name, threads, mm, data_type, st):
    out_path = os.path.join(wk_dir, '%s-%s-mm.bam' % (read_name, ref_name))
    if data_type == 'ont':
        x = 'map-ont'
    elif data_type == 'pacbio':
        x = 'map-pb'
    else:
        x = ''
    process = Popen([mm + ' -t ' + str(threads) + ' -ax ' + x + ' ' + ref + ' ' + read + ' | ' + st +
                     ' view -Sb - -o ' + out_path],
                    universal_newlines=True, stdout=PIPE, stderr=STDOUT, shell=True, executable='/bin/bash')
    with process.stdout:
        log_subprocess(process.stdout)
    exitcode = process.wait()
    if exitcode != 0:
        logging.critical("Error: minimap2 alignment failed")
        raise Exception("Error: minimap2 alignment failed, see log")
    return ['minimap2 -t ' + str(threads) + ' -ax ' + x + ' ' + ref + ' ' + read + ' | samtools view -Sb - -o ' + out_path,
            out_path]
```
My personal suggestion will be:
```
    if data_type == 'ont':
        x = 'x map-ont'
    elif data_type == 'pacbio-clr':
        x = 'x map-pb'
    elif data_type == 'pacbio-ccs':
        x = 'x map-hifi'
    else:
        x = ''
    process = Popen([mm + ' -t ' + str(threads) + ' -a' + x + ' ' + ref + ' ' + read + ' | ' + st +
                     ' view -Sb - -o ' + out_path],
                    universal_newlines=True, stdout=PIPE, stderr=STDOUT, shell=True, executable='/bin/bash')
```
 since the preset for minimap2 is like:
```
Preset:
    -x STR       preset (always applied before other options; see minimap2.1 for details) []
                 - map-pb/map-ont - PacBio CLR/Nanopore vs reference mapping
                 - map-hifi - PacBio HiFi reads vs reference mapping
                 - ava-pb/ava-ont - PacBio/Nanopore read overlap
                 - asm5/asm10/asm20 - asm-to-ref mapping, for ~0.1/1/5% sequence divergence
                 - splice/splice:hq - long-read/Pacbio-CCS spliced alignment
                 - sr - genomic short-read mapping
```

**Comment by cytham** [comment]
Hi @LYC-vio 

Thanks for identifying the issue. Seems like a bug. I will fix it in a while. Thanks!

**Comment by LYC-vio** [comment]
Hi @cytham 

I've created a pull request correspond to this issue

**Comment by cytham** [comment]
Thanks @LYC-vio. Fixed #37
